src/lib.rs

```diff
@@ -247,6 +248,12 @@ pub struct Config {
 	///
 	/// **Note:** A minimum of 10 seconds is always enforced.
 	pub fee_rate_cache_update_interval_secs: u64,
+	/// A list of peers which we allow to establish zero confirmation channels to us.
+	///
+	/// **Note:** Allowing payments via zero-confirmation channels is potentially insecure if the
+	/// funding transaction ends up never being confirmed on-chain. Zero-confirmation channels
+	/// should therefore only be accepted from trusted peers.
+	pub peers_trusted_0conf: Vec<PublicKey>,
```

Comment:
How about `trusted_peers_0conf` for consistency with `accept_inbound_channel_from_trusted_peer_0conf`